src/components/[guild]/forms/FillForm.tsx

```diff
@@ -129,6 +138,27 @@ const FillForm = ({ form }: Props) => {
   )
 }
 
+const FillFormWrapper = ({ form }: FormProp) => {
+  const { userSubmission, isLoading } = useUserFormSubmission(form)
+
+  if (!form || isLoading) return <FillFormSkeleton />
+
+  if (!userSubmission) return <FillForm form={form} />
+
+  if (form.isEditable)
+    return (
+      <FillForm
+        form={form}
+        method="PUT"
+        defaultValues={
+          userSubmission && mapAnswersToObject(userSubmission.submissionAnswers)
```

Comment:
I'd just put this inside the if condition above, just to make sure we don't ever render the edit form component without default values